src/SprykerEco/Zed/Heidelpay/Business/Adapter/Mapper/RequestToHeidelpay.php

```diff
@@ -48,15 +48,15 @@ public function map(HeidelpayRequestTransfer $requestTransfer, Request $heidelpa
 
         $heidelpayRequest->basketData(
             $requestTransfer->getCustomerPurchase()->getIdOrder(),
-            $requestTransfer->getCustomerPurchase()->getAmount(),
+            (string)$requestTransfer->getCustomerPurchase()->getAmount(),
```

Comment:
I'm not sure if it's safe enough. Which cases could be here?

Reply:
I suppose that (string) was added for normal float support inside JSON. Also if request params will not be correct - Heidelpay API will respond with error